### PR TITLE
object detection demo models

### PR DESCRIPTION
**Pull request by vladimir-dudnik**
add efficientdets and faceboxes to models.lst and test cases

### CODE REVIEW

File: demos/tests/cases.py [hunk 1]
```diff
@@ -237,8 +240,13 @@ def single_option_cases(key, *args):
             *combine_cases(
                 TestCase(options={'-at': 'yolo'}),
                 single_option_cases('-m',
+                    ModelArg('person-vehicle-bike-detection-crossroad-yolov3-1020'),
                     ModelArg('yolo-v3-tf'),
                     ModelArg('yolo-v3-tiny-tf'))),
+            *combine_cases(
+                TestCase(options={'-at': 'faceboxes'}),
+                single_option_cases('-m',
+                    ModelArg('faceboxes-pytorch'))),
```
Comment (Wovchena) [inline demos/tests/cases.py:249]: There are no other `faceboxes` models. You should merge `-at` and `-m` here.

Comment (vladimir-dudnik) [reply demos/tests/cases.py]: ok, changed. also put options in alphabet order

Comment (Wovchena) [reply demos/tests/cases.py]: Wrapping with `*combine_cases()` is not needed: product of one element is just that element.

Comment (Wovchena) [reply demos/tests/cases.py]: And as the test says there is no `--architecture_type` for C++ version, only `-at`.

Comment (vladimir-dudnik) [reply demos/tests/cases.py]: fixed

Comment (Wovchena) [reply demos/tests/cases.py]: There is `combine_cases()` left for python

**Review by Wovchena — APPROVED** [review]

**Pull request merged**